internal/cmd/config/config.go

```diff
@@ -57,6 +57,13 @@ kms "aead" {
 	key_id = "global_worker-auth"
 }
 
+kms "aead" {
+    purpose = "worker-storage"
```

Comment:
Similar question here. Would we define different keys for worker auth storage and other future storage needs? Possibly it should be a purpose of `worker-auth-storage`.

Reply:
So to confirm, we would like to change this WorkerStorageKms wrapper purpose and all its related value names to WorkerAuthStorageKms-flavored things, like I've done with the storage path names in dev.go?